tasks/interactive-bots.md

```diff
@@ -80,7 +78,7 @@ Follow the tutorial below to test run the followup bot on your Zulip instance
 
   Make sure to point to your `.zuliprc-local` file - if you have created it in
   a different folder than `~`, you have to point to the right folder, e.g.
-  `../../.zuliprc-local`.
+  `<folder-name>/<folder-name>/.zuliprc-local`.
```

Comment:
I don't agree with this change - this is an appropriate usage of the Terminal meaning "this file is two folders up".